invoke/runners.py

```diff
@@ -98,7 +98,9 @@ def run(self, command, **kwargs):
             the ``echo`` keyword, etc). The base default values are described
             in the parameter list below.
 
-        :param str command: The shell command to execute.
+        :param str|list command:
```

Comment:
`str|list` is what seems to fail CI - I have to figure out what syntax sphinx expects here.